src/main/java/Duke.java

```diff
@@ -45,79 +46,90 @@ public void handleError() {
         System.out.println(errorMessage);
     }
 
-    public void handleRequests(String userInput) {
-        if (userInput.startsWith("mark")) {
-            words = userInput.split(" ");
-            int posOfTask = Integer.parseInt(words[1]) - 1;
-            if (!(posOfTask >= 0 && posOfTask <= currentTask)) {
-                handleError();
+    public void printTotalTasks(int currentTaskNum) {
+        if (currentTaskNum == 1) {
+            System.out.println("Now you have " + currentTaskNum + " task in the list.");
+        } else {
+            System.out.println("Now you have " + currentTaskNum + " tasks in the list.");
+        }
+    }
+
+    public int getMarkPosition(String userInput) {

```

Comment:
I think you did a great job on the naming of variables and methods, they comply with the coding standard.